src/boxed.rs

```diff
@@ -214,14 +214,14 @@ macro_rules! glib_boxed_wrapper {
 
                 let mut res = Vec::with_capacity(num);
                 for i in 0..num {
-                    res.push($crate::translate::from_glib_none(ptr::read(ptr.add(i))));
+                    res.push($crate::translate::from_glib_none(::std::ptr::read(ptr.add(i))));
                 }
                 res
             }
 
             unsafe fn from_glib_container_num_as_vec(ptr: *mut *mut $ffi_name, num: usize) -> Vec<Self> {
                 let res = $crate::translate::FromGlibContainerAsVec::from_glib_none_num_as_vec(ptr, num);
-                glib_ffi::g_free(ptr as *mut _);
+                $crate::ffi::g_free(ptr as *mut _);
```

Comment:
Seems this not works in other crates

Reply:
See appveyour logs, I don't think that it fails in `ffi::g_tls_file_database_get_type(),`.
```
error[E0603]: module `ffi` is private
  --> C:\Users\appveyor\.cargo\git\checkouts\gio-a4b825b310a073b9\d1f28c4\src\auto\tls_file_database.rs:25:1
   |
25 | / glib_wrapper! {
26 | |     pub struct TlsFileDatabase(Object<ffi::GTlsFileDatabase, ffi::GTlsFileDatabaseInterface>): TlsDatabase;
27 | |
28 | |     match fn {
29 | |         get_type => || ffi::g_tls_file_database_get_type(),
30 | |     }
31 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```
also many errors in travis but its don't show from here it
```
error[E0603]: extern crate `ffi` is private
  --> <::glib::boxed::glib_boxed_wrapper macros>:78:24
   |
78 | let v_ptr = $ crate :: ffi :: g_malloc0 (
   |                        ^^^
```

Reply:
.. so my assumption that `extern crate glib_sys as ffi;` don't create exported module

Reply:
Ah right. This needs the boxed PR. That one is reexporting ffi

Reply:
I missed that you added `pub` to it and didn't understand why `#[doc(hidden)]` added

Reply:
.. now it make sense.
Then we wait until #397 merged